Desktop/utilities/qmlcacheconfiguration.h

```diff
@@ -0,0 +1,18 @@
+#ifndef QMLCACHECONFIGURATION_H
+#define QMLCACHECONFIGURATION_H
+
+#ifdef linux
+#include <QDir>
+
+class QMLCacheConfiguration
+{
+public:
+    static void configureQMLCacheDir();
```

Comment:
Also I think it might be better to add these functions to `Desktop/utilities/qmlutils.h/cpp`

Reply:
I believe the changes are as specified.